src/app/@core/services/crypto.service.ts

```diff
@@ -1,7 +1,10 @@
 import { Injectable } from '@angular/core';
 import { Logger } from '@core/services/logger.service';
 import { ElectronUtils } from '@core/utils/electron-utils';
-import { NFTCollectionAsset } from '@protokol/nft-base-crypto/dist/interfaces';
+import {
+  NFTCollectionAsset,
+  NFTTokenAsset,
+} from '@protokol/nft-base-crypto/dist/interfaces';
```

Comment:
Please try to avoid this in general and import interfaces from nft-base-crypto. I think I noticed this also in other code sections